### DISCUSSION TITLE
Receiver crashes on certain images

### DISCUSSION THREAD

**Discussion by ChrisM246** [dl9rdz/rdz_ttgo_sonde#336] [TTGO hardware]
Hello everyone, my TTGO V1.0 in connection with the ILI9225 crashes in pictures 4-13. Operation via the web portal still works, but the image on the display is frozen, only a restart helps.

**Comment by dl9rdz** [comment]
Hi, can you share your config file with me (http://rdzsonde/file/config.txt)?

You did not activate the "Display saver" feature? If you did, set it to 0 and retry and check if this happens as well.

What you can try as well is decreasing the SPI speed (e.g. to half of its value).

**Comment by ChrisM246** [comment]
Hi this is my config file. 
wifi=3
mdnsname=rdzsonde
ephftp=igs.bkg.bund.de/IGS/BRDC/
debug=0
maxsonde=20
rxlat=
rxlon=
rxalt=
screenfile=0
display=0,1,2,3,4,5,6,7,8,9,10,11,12,13,14
dispsaver=0
dispcontrast=-1
norx_timeout=20
tft_orient=1
spectrum=-1
startfreq=400
channelbw=10
marker=1
noisefloor=-125
freqofs=0
rs41.agcbw=12500
rs41.rxbw=6300
rs92.rxbw=12500
rs92.alt2d=480
dfm.agcbw=20800
dfm.rxbw=10400
m10m20.agcbw=20800
m10m20.rxbw=12500
mp3h.agcbw=12500
mp3h.rxbw=12500
call=N0CALL
passcode=12345
kisstnc.active=1
axudp.active=1
axudp.host=192.168.42.20
axudp.port=9002
axudp.highrate=1
tcp.active=0
tcp.host=radiosondy.info
tcp.port=14590
tcp.highrate=20
tcp.objcall=
tcp.beaconsym=/`/(
tcp.chase=0
tcp.comment=
mqtt.active=1
mqtt.id=rdz_sonde_server
mqtt.host=192.168.1.16
mqtt.port=1883
mqtt.username=
mqtt.password=
mqtt.prefix=rdz_sonde_server/
cm.active=0
cm.host=
cm.port=0
sondehub.active=1
sondehub.chase=0
sondehub.host=api.v2.sondehub.org
sondehub.callsign=DC1MU
sondehub.antenna=Diamond X300
sondehub.email=DC1MU@darc.de
sondehub.fiactive=0
sondehub.fiinterval=5
sondehub.fimaxdist=150
sondehub.fimaxage=2
disptype=1
oled_sda=21
oled_scl=22
oled_rst=13
tft_rs=4
tft_cs=0
tft_spifreq=40000000
button_pin=35
button2_pin=25
button2_axp=1
touch_thresh=70
power_pout=-1
led_pout=-1
gps_rxd=34
gps_txd=-1
batt_adc=-1
sx1278_ss=18
sx1278_miso=19
sx1278_mosi=27
sx1278_sck=5

**Comment by m1cdq** [comment]
I am having the same issue  TTGO t-Beam 1.1  running on latest dev firmware with a TFT ILI9341 display.
It locks up on screen 4,5,6,7,8,9,10,11,12,13.  control via webpage still works. just the display freeze up

My config.txt  Ive have even tried  different TFT SPI speeds  and screen orientations 
wifi=3
mdnsname=rdzsonde2
ephftp=gssc.esa.int/gnss/data/daily/
debug=0
maxsonde=40
rxlat=53.5136
rxlon=-0.10892
rxalt=12
screenfile=0
display=0,1,2,3,12
dispsaver=0
dispcontrast=-1
norx_timeout=20
tft_orient=1
spectrum=-1
startfreq=400
channelbw=10
marker=1
noisefloor=-125
freqofs=0
rs41.agcbw=12500
rs41.rxbw=6300
rs92.rxbw=12500
rs92.alt2d=480
dfm.agcbw=20800
dfm.rxbw=10400
m10m20.agcbw=20800
m10m20.rxbw=12500
mp3h.agcbw=12500
mp3h.rxbw=12500
call=M1CDQ
passcode=12345
kisstnc.active=1
axudp.active=1
axudp.host=192.168.42.20
axudp.port=9002
axudp.highrate=1
tcp.active=0
tcp.host=radiosondy.info
tcp.port=14590
tcp.highrate=20
tcp.objcall=
tcp.beaconsym=/`/(
tcp.chase=0
tcp.comment=
mqtt.active=0
mqtt.id=rdz_sonde_server
mqtt.host=
mqtt.port=1883
mqtt.username=
mqtt.password=
mqtt.prefix=rdz_sonde_server/
cm.active=0
cm.host=
cm.port=0
sondehub.active=1
sondehub.chase=3
sondehub.host=api.v2.sondehub.org
sondehub.callsign=M1CDQ/M2
sondehub.antenna=1/4 wave
sondehub.email=dwabbit@ntlworld.com
sondehub.fiactive=1
sondehub.fiinterval=5
sondehub.fimaxdist=300
sondehub.fimaxage=24
disptype=3
oled_sda=21
oled_scl=22
oled_rst=16
tft_rs=4
tft_cs=15
tft_spifreq=15000000
button_pin=39
button2_pin=13
button2_axp=0
touch_thresh=70
power_pout=-1
led_pout=-1
gps_rxd=12
gps_txd=-1
batt_adc=-1
sx1278_ss=18
sx1278_miso=19
sx1278_mosi=27
sx1278_sck=5

**Comment by m1cdq** [comment]
I think i found the problem and a temp fix 
I now have screens 0,1,2,3,6,8,9,10,11,12 running
on the pages that jammed  all were asking for info on battery voltage in the footer
ie  **_12.7,18=bVV_**
i am using ttgo board v1.1 
all i have done is  put a # at those lines  to null them out 
here is my screens4.txt 

## screens2.txt: TFT display (landscape)
# Definition of display content and action behaviour
#
# Timer: (view timer, rx timer, norx timer)
# - value -1: timer is disabled; value>=0: timer fires after (value) seconds
# - view timer: time since current view (display mode and sonde) was started
# - rx timer: time since when sonde data has been received continuously (trigger immediatly after RX)
# - norx timer: time since when no sonde data has been received continuously
#   (rx and norx timer is started after tuning a new frequency and receiving a signal or not receiving
#    anything for a 1s period)
# 
# Actions:
# - W: activate WiFi scan
# - F: activate frequency spectrum display
# - 0: activate "Scan:" display (this is basically just display mode 0)
# - x: (1..N): activate display mode x [deprecated]
# - >: activate next display mode 
# - D: activate default receiver display (display mode specified in config)
# - +: advance to next active sonde from QRG config
# - #: no action
#
# Display content (lower/upper case: small/large font)
#    line,column=content
# for ILI9225 its also possible to indicate
#    line,column,width=content          for text within a box of width 'width'
#    line,column,-width=content         for right-justified text
#
# XText : Text
# F(suffix): frequency (with suffix, e.g., " MHz")
# L latitade
# O lOngitute
# A altitude
# Hm(suffix) hor. speed m/s (suffix: e.g. "m/s"; no suffix=>m/s as 16x8 bitmap for SSD1306 display only)
# Hk(suffix) hor. speed km/h (suffix: e.g. "km/h"; no suffix=>km/h as 16x8 bitmap for SSD1306 display only)
# V(suffix) vert. speef (suffix: e.g. "m/s"; no suffix=>m/s  as 16x8 bitmap for SSD1306 display only)
# Ix sonde ID (default/d: dxlaprs; s: short id, n: real serial number)
#    RS41,RS92: all identical R1234567
#    DFMx: ID M12345678; short ID and serial 12345678
#    M10: ID ME95231F0; short ID: M95231F0; serial 9062104592
# Q signal quality statistics bar
# T type string (RS41/DFM9/DFM6/RS92)
# C afC value
# N ip address (only tiny font)
# S scan list entry info:  l/empty: launch site name, #=entry nr, t=total entries, a=active entries, /: #/t  
# K RS41 kill timer values: Kl launch timer, Kb burst timer, Kc kill countdown
#   format: K_4: h:mm k_6: h:mm:ss k_s: sssss,  nothing shown for other sonde
# Mx telemetry value x (t temp p preassure h hyg) [not yet implemented, maybe some day in future]
# Gx  GPS-related data
#     raw data from GPS:  GA, GO, GH, GC: LAtitude, lOngitude, Altutide(Height), Course over ground
#     relative to sonde:  GD, GI, GB: Distance, dIrection (absolute), relative Bearing
# G0  GPS circle diagram e.g.  3,5=g0NCS,50,ff0000,000033,5,ffff00,4,ffffff
#     "N" (what is on top: N=north C=course)
#     "C" (where does the arrow point to: C=course, S=sonde)
#     "S" (what is shown by the bullet: C=course, S=sonde)
#     50: circle radius, followed by fg and bg color
#     5: bullet radius, followed by fg color
#     4: arrow width, followed by fg color
# R RSSI
# B battery(T-Beam 1.0) S=status V=Batt.Volt C=charge current D=discharge current
#                       U=USB volt I=USB current  T=IC temp
#
# fonts=x,y can be used to select font (x=small, y=large) for all items below
# for SSD1306, x and y can be used to select one of those fonts:
# (y should be a 1x2 font (1,5,6,7), x a small font)
#                u8x8_font_chroma48medium8_r,        // 0 ** default small
#                u8x8_font_7x14_1x2_f,               // 1 ** default large
#                u8x8_font_amstrad_cpc_extended_f,   // 2
#                u8x8_font_5x7_f,                    // 3
#                u8x8_font_5x8_f,                    // 4
#                u8x8_font_8x13_1x2_f,               // 5
#                u8x8_font_8x13B_1x2_f,              // 6
#                u8x8_font_7x14B_1x2_f,              // 7
#                u8x8_font_artossans8_r,             // 8
#                u8x8_font_artosserif8_r,            // 9
#                u8x8_font_torussansbold8_r,         // 10 
#                u8x8_font_victoriabold8_r,          // 11
#                u8x8_font_victoriamedium8_r,        // 12
#                u8x8_font_pressstart2p_f,           // 13
#                u8x8_font_pcsenior_f,               // 14
#                u8x8_font_pxplusibmcgathin_f,       // 15
#                u8x8_font_pxplusibmcga_f,           // 16
#                u8x8_font_pxplustandynewtv_f,       // 17
#
# for ILI9225, these fonts are available:
# 	Terminal6x8		// 0
#       Terminal11x16		// 1
#       Terminal12x16		// 2
#	FreeMono9pt7b,          // 3
#       FreeMono12pt7b,         // 4
#       FreeSans9pt7b,          // 5
#       FreeSans12pt7b,         // 6
#       Picopixel,  		// 7
#
# color=rrggbb,rrggbb  can be used to select color (foreground, background)
# see https://github.com/Nkawu/TFT_22_ILI9225/wiki#color-reference for example (use without "#"-sign)
#
# for TFT display, coordinates and width are multiplied by xscale,yscale and later used in pixels
# with scale=1,1 you can directly use pixel coordinates.  (default: xscale=13,yscale=22 => 8 lines, 16 columns)
###########

############
# Scan display for large 2" TFT dispaly
@ScannerTFT
scale=30,18
timer=-1,0,0
key1action=D,#,F,W
key2action=D,#,#,#
timeaction=#,D,+
fonts=5,6
0,0=XScan
0,5,-3=S#:
0,9,5.5=T
3,0=F MHz
5,0,16=S
7,5=n

############
@MainTFT
timer=-1,-1,N
key1action=+,0,F,W
key2action=>,#,#,#
timeaction=#,#,0
color=FFD700
0,0,10.5=Is
color=0000FF
0,11,-5.5=f
1,0,4=t
1,10.5,-6=c
color=00ff00
2,0,7=L
4,0,7=O
color=FFA500
2,9.5,-7=A
3,9.5,-7=vm/s
color=AA5522
4,9.5,-7=hkkm/h
color=FFFFFF
6,2=r
6.3,10=Q4
7,0=xd=
7,2,6=gD
7,12=gI

############
@PeilungTFT
timer=-1,-1,N
key1action=+,0,F,W
key2action=>,#,#,#
timeaction=#,#,#
color=ffff00,000033
color=bbbbbb,000000
0,2=xN Top:
0,8=xCourse Top:
color=ffff00,000033
1,0=g0NCS,48,ffff00,000044,6,33ff33,5,eeaa00
1,8=g0CCS,48,ffff00,000044,6,55ff55,5,eeaa00
color=ffffff,000000
6,0=xDirection:
6,8,4=gI
7,0=xCOG:
7,4,4=gC
7,8=xturn:
7,12,4=gB

############
@GPSdataTFT
timer=-1,-1,N
key1action=+,0,F,W
key2action=>,#,#,#
timeaction=#,#,#
0,0=xOn-board GPS:
1,0,8=gA
2,0,8=gO
3,0,8=gH
4,0,8=gC
5,0=xGPS vs Sonde:
6,0,8=gD
7,0,8=gI
7,8,8=gB

############
@BatteryTFT
timer=-1,-1,-1
key1action=+,0,F,W
key2action=>,#,#,#
timeaction=#,#,#
0,0=xBattery status:
0,14=bS
1,0=xBatt:
#1,5,5=bVV
2,0,16=bCmA(charging)
3,0,16=bDmA(discharging)
4.4,0=xUSB:
4.4,5,5=bUV
5.4,0,10=bImA
6.4,0=xTemp:
6.4,5,5=bT C

### Alternative display layouts based on https://gist.github.com/bazjo 
# Scan display for large 2" TFT dispaly
@Scan.TFT.Bazjo
timer=-1,0,0
key1action=D,#,F,W
key2action=D,#,#,#
timeaction=#,D,+
scale=11,10
fonts=0,2
color=e0e0e0
#Row 1
0.5,0=XScanning...
#Row 2
3,0=xIndex
4,0,8=S/
3,9=xSite
4,9=S
#Row 3
6,0=xType
7,0,6=T
6,9=xFrequency
7,9=F
#Row 4
9,0=xWeb UI IP
10,0=N
#Row 5
#Footer
color=6C757D
15,0=xScan Mode
#15,18=bVV

############
@Decode/General.TFT.Bazjo
timer=-1,-1,N
key1action=+,0,F,W
key2action=>,#,#,#
timeaction=#,#,0
scale=11,10
fonts=0,2
#Row 1
color=996A06
0,0=xSerial
0,5=t
color=FFB10B
1,0=Is
color=996A06
0,11=xFreq.
0,16=c
color=FFB10B
1,11=F
#Row 2
color=3C5C99
3,0=xLatitude
color=639AFF
4,0=L
color=3C5C99
3,11=xLongitude
color=639AFF
4,11=O
#Row 3
color=3C5C99
6,0=xHoriz. Speed
color=639AFF
7,0=Hkkm/h
color=3C5C99
6,11=xVert. Speed
color=639AFF
7,11=Vm/s
#Row 4
color=99004A
9,0=xAltitude
color=FF007B
10,0=A
color=99004A
9,11=xBearing
color=FF007B
10,11=GB
#Row 5
color=06998E
12,0=xRSSI
color=0AFFEF
13,0=R
color=06998E
12,11=xHistory
color=0AFFEF
13.5,11=Q4
#Footer
color=6C757D
15,0=xDecode Mode / General View
#15,18=bVV

############
@Decode/Battery.TFT.Bazjo
timer=-1,-1,N
key1action=+,0,F,W
key2action=>,#,#,#
timeaction=#,#,#
scale=11,10
fonts=0,2
#Row 1
color=99001F
0,0=xBattery Status
0,11=xBattery Voltage
color=FF0035
1,0=BS
#1,11=BVV
#Row 2
color=99001F
3,0=xCharge Current
3,11=xDischarge Current
color=FF0035
4,0=BCmA
4,11=BDmA
#Row 3
color=99001F
6,0=xUSB Voltage
6,11=xUSB Current
color=FF0035
7,0=BUV
7,11=BImA
#Row 4
color=99001F
9,0=xIC Temperature
#9,11=xKey
color=FF0035
10,0=BTC
#10,11=XValue
#Row 5
#12,0=xKey
#12,11=xKey
#13,0=XValue
#13,11=XValue
#Footer
color=99001F
15,0=xDecode Mode/Battery View
#15,18=bVV

# based on https://github.com/puspis/rdz_ttgo_sonde
##########
@Scanner.Puspis
timer=-1,0,4
key1action=D,#,F,W
key2action=D,#,#,#
timeaction=#,D,+
scale=13,10
fonts=0,1
#Row 1
color=90EE90
0.5,3=XFREQUENCY SCAN
#Row 2
color=00FF00
3,0=xMEMORY
3,9=xLAUNCH SITE
color=639AFF
4,0,9=S/
4,9=S
#Row 3
color=00FF00
6,0=xTYPE
6,9=xFREQUENCY
color=639AFF
7,0,9=T
7,9=F MHz
#Row 4
fonts=0,5
color=285454
11.5,0=xIP ADDRESS:
10.9,7,-15=N
#Footer
color=FF0000
#12.7,18=bVV

############
@Main.Puspis
timer=-1,-1,N
key1action=+,0,F,W
key2action=>,#,#,#
timeaction=#,#,0
scale=11,10
fonts=0,2
#Row 1
color=00FF00
0,0=xSONDE ID
0,11=xFREQUENCY
color=90EE90
0,7.4=t
1,0=Is
1,11=F
#Row 2
fonts=0,1
color=00FF00
3,0=xLATITUDE
3,11=xLONGITUDE
color=FF007B
4,0=L
4,11=O
#Row 3
color=00FF00
6,0=xWIND SPEED
6,11=xCLIMB RATE
color=639AFF
7,0=Hkkm/h
7,11=Vm/s
#Row 4
color=00FF00
9,0=xRX ALTITUDE
9,11=xSONDE ALTITUDE
color=639AFF
10,0=GH
10,11=A
#Row 5
color=00FF00
12,0=xDISTANCE
12,11=xFRAMES
color=FFFFFF
13,0=GD
13.5,11=Q4
#Footer
color=FF0000
15,0.2=xIP:
15,2.5=n
#15,18=bVV

############
@JotaEme.Puspis
timer=-1,-1,N
key1action=+,0,F,W
key2action=>,#,#,#
timeaction=#,#,0
scale=11,10
fonts=0,1
#Row 1
color=90EE90
0,0=Is
0.7,10.5=t
0,14=F
#Row 2
color=00FF00
2,1.2=xWIND SPEED
2,14=xCLIMB RATE
color=639AFF
3,1=Hkkm/h
3,13=Vm/s
#Row 3
color=00FF00
5,1=xRX ALTITUDE
5,12.5=xSONDE ALTITUDE
color=639AFF
6,2=GH
6,14=A
#Row 4
color=00FF00
8,0=xSONDE POSITION
color=FF007B
9,2=l
10,2=o
#Row 5
color=00FF00
11.4,2=xDISTANCE
color=FFFFFF
12.4,1.5=GD
#Circle
color=EEAA00,000033
8,13.8=g0CCS,28,FFFF00,000033,5,9ACD32,5,EEAA00
#Footer
color=FF0000
15,0=n
color=FFFFFF,000000
15,8.7=Q4
color=FF0000
#15,18.4=bVV

############
@CompassTFT.Puspis
timer=-1,-1,N
key1action=+,0,F,W
key2action=>,#,#,#
timeaction=#,#,#
scale=13,10
fonts=0,1
#Row 1
color=90EE90
0.5,1.5=XCOMPASS
#Row 2
color=00FF00
4,2=xRX HEADING
color=639AFF
5,3.8=GC
#Row 3
color=00FF00
9.5,3=xDISTANCE
9.5,13.5=xBEARING
color=639AFF
10.5,3.4=GD
10.5,14.3=GI
#Circle
color=EEAA00,000033
0.2,10=g0CCS,52,FFFF00,000033,10,9ACD32,6,EEAA00
#Footer
color=FF0000,000000
12.7,1=Q4
#12.7,18=bVV

############
@GPSdataTFT.Puspis
timer=-1,-1,N
key1action=+,0,F,W
key2action=>,#,#,#
timeaction=#,#,#
scale=13,10
fonts=0,1
#Row 1
color=90EE90
0.5,0.5=XGPS RECEIVER STATION
#Row 2
color=00FF00
3,0=xRX LATITUDE
3,12=xRX LONGITUDE
color=639AFF
4,0=GA
4,12=GO
#Row 3
color=00FF00
6,0=xRX ALTITUDE
6,12=xRX HEADING
color=639AFF
7,0=GH
7,12=GC
#Row 4
color=00FF00
9,0=xDISTANCE
9,12=xBEARING
color=639AFF
10,0=GD
10,12=GB
#Footer
color=FF0000,000000
12.7,0.4=Q4
#12.7,18=bVV

############
@BatteryTFT.Puspis
timer=-1,-1,-1
key1action=+,0,F,W
key2action=>,#,#,#
timeaction=#,#,#
scale=13,10
fonts=0,1
#Row 1
color=90EE90
0.5,4=XBATTERY STATUS
#Row 2
color=00FF00
3,0=x(C)HARGE/(B)ATT
3,11.5=xBATTERY VOLTAGE
color=639AFF
4,4=BS
4,11.5=BVV
#Row 3
color=00FF00
6,0=xCHARGE CURRENT
6,11.5=xDISCHG. CURRENT
color=639AFF
7,0=BCmA
7,11.5=BDmA
#Row 3
color=00FF00
9,0=xIC TEMPERATURE
9,11.5=xFREQ. OFFSET
color=639AFF
10,0=BTC
10,10=C
#Footer
color=FF0000,000000
12.7,0.4=Q4
#12.7,18=bVV
